include/autoppl/expression/distribution/uniform.hpp

```diff
@@ -0,0 +1,50 @@
+#pragma once
+#include <cassert>
+#include <random>
+#include <autoppl/util/dist_expr_traits.hpp>
+#include <autoppl/util/var_expr_traits.hpp>
+#include <autoppl/expression/distribution/base.hpp>
+
+namespace ppl {
+namespace expr {
+
+template <typename min_type, typename max_type>
+struct Uniform
+{
+    static_assert(util::is_var_expr_v<min_type>);
+    static_assert(util::is_var_expr_v<max_type>);
+
+    using value_t = util::cont_raw_param_t;
+    using param_value_t = std::common_type_t<
```

Comment:
Is there any distribution where we have parameters of different types?

Reply:
Binomial.